test/webview/SampleView.test.ts

```diff
@@ -273,6 +292,35 @@ describe('SampleView', () => {
             samplePageHtml.should.contain(`<button class="open-button" onclick="openFile('applications','JavaScript Application')">Open Locally</button>`); // Disabled open button
             samplePageHtml.should.contain(`<button class="open-button" onclick="openFile('applications','TypeScript Application')">Open Locally</button>`); // Disabled open button
         });
+
+        it('should get correct html when cloned and there is no internet connection', async () => {
+
+            const error: Error = new Error('error happened');
+            mySandBox.stub(Axios, 'get').rejects(error);
+            mySandBox.stub(ejs, 'renderFile').callThrough();
+
+            const readFileSyncStub: sinon.SinonStub = mySandBox.stub(fs, 'readFileSync').resolves('README read here');
+
+            await RepositoryRegistry.instance().clear();
+
+            const repoEntry: RepositoryRegistryEntry = new RepositoryRegistryEntry();
```

Comment:
i thought you were going to put this in the beforeEach

Reply:
I was planning to, but its only used twice - is it still worth doing it?

Reply:
yes to get rid of the `repositoryGetStub` thats used later

Reply:
Fixed - Removed the repositoryRegistryGetStub